api/scripts/wazuh_apid.py

```diff
@@ -5,6 +5,8 @@
 # This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
 
 import argparse
+from concurrent.futures import ProcessPoolExecutor, ThreadPoolExecutor
+from functools import partial
 import os
 import signal
 import sys
```

Comment:
To be PEP8 compliant 

```suggestion
import argparse
import os
import signal
import sys
import warnings
from concurrent.futures import ProcessPoolExecutor, ThreadPoolExecutor
from functools import partial
```